client/state/initial-state.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import debugModule from 'debug';
+import localforage from 'localforage';
+
+/**
+ * Internal dependencies
+ */
+import { createReduxStore, reducer } from 'state';
+import { SERIALIZE, DESERIALIZE } from 'state/action-types'
+import config from 'config';
+
+/**
+ * Module variables
+ */
+const debug = debugModule( 'calypso:state' );
+
+export const localforageConfig = {
+	name: 'calypso',
+	storeName: 'calypso_store',
+	description: 'Calypso Storage',
+	driver: localforage.INDEXDB
```

Comment:
This should be `driver: localforage.INDEXEDDB`

Probably works only because the default assumes IndexedDB:

https://github.com/mozilla/localForage/blob/f473d39943bc6536bb42eb2587f61d685a7a6f76/dist/localforage.js#L765